projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -71,6 +72,14 @@ export class GoTableComponent implements OnInit, OnChanges {
     this.renderTable();
   }
 
+  ngAfterContentInit(): void {
```

Comment:
If we default sorting to `true` on a column, then we won't have to worry about any of this.

Reply:
@grahamhency We can default sorting to true, but not defaulting it allows us to have a little more flexibility in how the table is implemented. This way instead of just turning sorting off for a couple columns, we can turn off sorting for the entire table and then enable sorting on a couple columns. The only way to do this is to always keep track of sorting on the column instead of the table, but if this isn't something we care about theres a few things I can clean up here that wouldn't be necessary.